exporter/clickhouseexporter/README.md

```diff
@@ -294,7 +294,17 @@ ClickHouse tables:
 
 - `logs_table_name` (default = otel_logs): The table name for logs.
 - `traces_table_name` (default = otel_traces): The table name for traces.
-- `metrics_table_name` (default = otel_metrics): The table name for metrics.
```

Comment:
Is `metrics_table_name` deprecated? Should we mention that on the changelog?

Reply:
should be adapted in the changelog